scylla/src/transport/errors.rs

```diff
@@ -34,15 +34,15 @@ use thiserror::Error;
 
 use crate::{authentication::AuthError, frame::response};
 
-use super::{
-    iterator::NextRowError,
-    legacy_query_result::IntoLegacyQueryResultError,
-    query_result::{IntoRowsResultError, SingleRowError},
-};
+use super::iterator::NextRowError;
+#[allow(deprecated)]
+use super::legacy_query_result::IntoLegacyQueryResultError;
+use super::query_result::{IntoRowsResultError, SingleRowError};
```

Comment:
Importing a deprecated item requires `#[allow(deprecated)]`?

Reply:
Indeed.

Reply:
You need that allow in every place where you name a deprecated item.

Reply:
Ok